Patch taken from
https://gitweb.gentoo.org/proj/musl.git/plain/x11-libs/vte/files/vte-0.54.2-musl-remove-W_EXITCODE.patch

Expanded non-standard W_EXITCODE macro for Musl compatibility

--- vte-0.62.1/src/widget.cc.~1~	2020-10-08 13:30:11.000000000 +0000
+++ vte-0.62.1/src/widget.cc	2020-11-22 15:52:21.405550775 +0000
@@ -20,8 +20,6 @@
 
 #include "widget.hh"
 
-#include <sys/wait.h> // for W_EXITCODE
-
 #include <exception>
 #include <new>
 #include <string>
@@ -235,7 +233,7 @@
 Widget::dispose() noexcept
 {
         if (m_terminal->terminate_child()) {
-                int status = W_EXITCODE(0, SIGKILL);
+                int status = (0) << 8 | (SIGKILL); // W_EXITCODE(ret, sig)
                 emit_child_exited(status);
         }
 }
